lib/fluent/plugin/map_support.rb

```diff
@@ -19,7 +19,7 @@ class MapSupport
     def initialize(map, plugin)
       @map = map
       @plugin = plugin
-      if defined?(Fluent::Filter) and plugin.is_a?(Fluent::Filter)
+      if defined?(Fluent::Plugin::Filter) and plugin.is_a?(Fluent::Plugin::Filter)
```

Comment:
`plugin.is_a?(Fluent::Plugin::Filter)` is enough.
Because we can drop old Fluentd (v0.12 or earlier) support.